src/netif.c

```diff
@@ -2528,60 +2569,24 @@ static void netif_lcore_init(void)
     lcore_role_init();
 
     /* register lcore jobs*/
-    netif_jobs[0].role = LCORE_ROLE_FWD_WORKER;
-    snprintf(netif_jobs[0].job.name, sizeof(netif_jobs[0].job.name) - 1, "%s", "recv_fwd");
-    netif_jobs[0].job.func = lcore_job_recv_fwd;
-    netif_jobs[0].job.data = NULL;
-    netif_jobs[0].job.type = LCORE_JOB_LOOP;
-
-    netif_jobs[1].role = LCORE_ROLE_FWD_WORKER;
-    snprintf(netif_jobs[1].job.name, sizeof(netif_jobs[1].job.name) - 1, "%s", "xmit");
-    netif_jobs[1].job.func = lcore_job_xmit;
-    netif_jobs[1].job.data = NULL;
-    netif_jobs[1].job.type = LCORE_JOB_LOOP;
-
-    netif_jobs[2].role = LCORE_ROLE_FWD_WORKER;
-    snprintf(netif_jobs[2].job.name, sizeof(netif_jobs[2].job.name) - 1, "%s", "timer_manage");
-    netif_jobs[2].job.func = lcore_job_timer_manage;
-    netif_jobs[2].job.data = NULL;
-    netif_jobs[2].job.type = LCORE_JOB_LOOP;
-
-    netif_jobs[3].role = LCORE_ROLE_ISOLRX_WORKER;
-    snprintf(netif_jobs[3].job.name, sizeof(netif_jobs[3].job.name) - 1, "%s", "isol_pkt_rcv");
-    netif_jobs[3].job.func = recv_on_isol_lcore;
-    netif_jobs[3].job.data = NULL;
-    netif_jobs[3].job.type = LCORE_JOB_LOOP;
-
-    netif_jobs[4].role = LCORE_ROLE_MASTER;
-    snprintf(netif_jobs[4].job.name, sizeof(netif_jobs[4].job.name) - 1, "%s", "timer_manage");
-    netif_jobs[4].job.func = lcore_job_timer_manage;
-    netif_jobs[4].job.data = NULL;
-    netif_jobs[4].job.type = LCORE_JOB_LOOP;
-
-    netif_jobs[5].role = LCORE_ROLE_MASTER;
-    snprintf(netif_jobs[5].job.name, sizeof(netif_jobs[5].job.name) - 1, "%s", "kni_master_proc");
-    netif_jobs[5].job.func = kni_process_on_master;
-    netif_jobs[5].job.data = NULL;
-    netif_jobs[5].job.type = LCORE_JOB_LOOP;
```

Comment:
Kni job has updated by pr #672  . Please fix the conflicts by spliting the kni job to two `dpvs_lcore_job` with different roles. `dpvs_lcore_job_register` could manage them correctly then.

Reply:
Introduce dpvs_job_job_init() to solve this issue within the new patch.